### ISSUE TITLE
Password restrictions

### ISSUE THREAD

**Issue by brussee** [csrdelft/balins-adventure#43]
zie ook: https://trac.knorrie.org/csrdelft.nl/ticket/549

Momenteel is nog elk password toegestaan.
- [ ] restricties
- [ ] ui integratie

**Comment by ajrouvoet** [comment]
Yeah, django password restrictions moeten sowieso nog worden geimplementeerd

**Comment by brussee** [comment]
Als je gebruikers helpt een goed wachtwoord te maken is dat beter dan van ze verwachten dat ze dat zelf kunnen, zie: https://csrdelft.nl/forum/reactie/96528#96528

**Comment by ajrouvoet** [comment]
Daarom heb ik bovenstaand beide onderdelen in een lijstje gezet

**Comment by thervh70** [comment]
Op dit moment ben ik wat research aan het doen naar mooie libraries voor dit probleem, omdat dit simpelweg niet gebeunt moet worden! Persoonlijk ben ik een groot voorstander van two-factor-authentication (https://github.com/Bouke/django-two-factor-auth)  maar ik weet niet of ik ieder lid kan verplichten dit te gebruiken, ivm het ontbreken van smartphones bij enkele leden waarmee tokens gegenereerd moeten worden.

Daarnaast heb ik ook deze library gevonden:https://github.com/dstufft/django-passwords. Dit is puur om makkelijk eisen te kunnen stellen aan de wachtwoord. Mijn idee is om dit te combineren met de TFA van hierboven. !(TFA) == meer eisen voor wachtwoord (langer bijv).

Graag wil ik van jullie horen of jullie deze libraries geschikt achten @ElessarWebb , @intel478
